tinytodo/src/policy_store.rs

```diff
@@ -49,23 +49,17 @@ impl PolicySetWatcher {
                     // This is less clean then I'd like, but modern editors seem to edit files in different ways.
                     // I've tested this w/ VSCode and Neovim.
                     // Closing a buffer in Neovim triggers the `Remove` event, require re-watching the file
-                    // Closing a file in VSCode triggers the `Close` event.
+                    // Closing a file in VSCode triggers the `Close` event on Linux, on MacOs it seems to send a `DatChange::Any`.
```

Comment:
Changing the sort of events to look for seems brittle.

Instead, can you just look at the file every second to see whether it’s been changed? I realize that per-second polling is not the most efficient thing to do, but it's simple and should give you reliable results. And this is just a demo.